project/npda/views/npda_users.py

```diff
@@ -288,19 +316,27 @@ def get_form_kwargs(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        context["show_rcpch_team_box"] = (
+            self.request.user.is_superuser
+            or self.request.user.is_rcpch_audit_team_member
```

Comment:
@mbarton @eatyourpeas we have a checkbox for is_rcpch_audit_team_member and also a possible role allocation. Do we need both, is there an instance in which someone would select reader/editor/coordinator and still be in the audit team, ticking that box? And what would be the rationale, as opposed to simply selecting the actual audit team role from the dropdown?